engine/src/main/java/org/destinationsol/game/SaveManager.java

```diff
@@ -52,17 +79,82 @@ public static void writeShip(HullConfig hull, float money, ArrayList<SolItem> it
                 }
             }
         }
-        IniReader.write(FILE_NAME, "hull", hullName, "money", (int) money, "items", sb.toString());
+
+        return sb.toString();
+
     }
 
-    public static boolean hasPrevShip() {
-        String path;
+    /**
+     * Writes the player's mercenaries to their JSON file.
+     * Will create file if it doesn't exist.
```

Comment:
`Will create file if it doesn't exist.` -> `The file will be created if it doesn't exist.`